data/remnant ships.txt

```diff
@@ -348,6 +347,87 @@ ship "Gull" "Gull (Bunks)"
 
 
 
+ship "Ibis"
+	sprite "ship/ibis"
+	thumbnail "thumbnail/ibis"
+	attributes
+		category "Medium Warship"
+		licenses
+			"Remnant Capital"
+		"cost" 30000000
+		"shields" 22000
+		"hull" 11000
+		"required crew" 20
+		"bunks" 48
+		"mass" 300
+		"drag" 7.1
+		"heat dissipation" 0.7
+		"fuel capacity" 700
+		"ramscoop" 2
+		"outfit space" 568
+		"cargo space" 103
+		"weapon capacity" 200
+		"engine capacity" 142
+		"shield generation" 6
+		"shield energy" 4.6
+		"hull repair rate" 2
+		"hull energy" 1.2
```

Comment:
More shield and hull generation than the Albatross, but at the same energy drain?

Reply:
Brought the numbers in much closer to the Albatrosses. Dropped all the values a little, since it isn't a heavy warship and doesn't have as much to regenerate. That being said, it is going to be a little more energy efficient. Not by much, though.